docs/pages/pmd/languages/java_metrics_index.md

```diff
@@ -23,6 +23,44 @@ it uses instead of the services they provide.
  
 ATFD can be used to detect God Classes and Feature Envy. \[[Lanza05](#Lanza05)\]
 
+## Class Fan Out Complexity (CFO)
+
+*Operation metric, class metric.* Can be computed on classes, enums and 
+concrete operations.
+
+### Description
+This counts the number of other classes a given class or operation relies on.
+Classes from the package `java.lang` are ignored by default (can be changed via options).
+Also primitives are not included into the count.
+
+### Code example
+
+```java
+import java.util.*;
+import java.io.IOException;
+
+public class Foo { // total 8
+    public Set set = new HashSet(); // +2
+    public Map map = new HashMap(); // +2
+    public String string = ""; // from java.lang -> does not count by default
+    public Double number = 0.0; // from java.lang -> does not count by default
+    public int[] intArray = new int[3]; // primitive -> does not count
+
+    @Deprecated // +1
+    @Override // +1
```

Comment:
Since Override is from java.lang, this shouldn't count...

Reply:
Thanks for the hint!